packages/admin-ui/ui/public/locales/en-US/translation.json

```diff
@@ -0,0 +1,1898 @@
+  "general.are-you-sure-you-want-to-delete-this-promotion": "Are you sure you want to delete this promotion?",
+  "general.promotion-deleted-successfully": "Promotion deleted successfully",
+  "Discount published successfully": "",
+  "Discount drafted successfully": "",
+  "general.delete-discount": "Delete discount",
+  "general.template-discount": "Template discount",
+  "general.published": "Published",
+  "general.draft": "Draft",
+  "general.discount-amount": "Discount Amount",
+  "general.valid-regions": "Valid Regions",
+  "general.total-redemptions": "Total Redemptions",
+  "FREE SHIPPING": "",
+  "general.unknown-discount-type": "Unknown discount type",
+  "details.discount-deleted": "Discount deleted",
+  "details.are-you-sure-you-want-to-delete-this-discount": "Are you sure you want to delete this discount?",
+  "details.delete-discount": "Delete discount",
+  "details.back-to-discounts": "Back to Discounts",
+  "details.raw-discount": "Raw discount",
+  "discounts.add-discount": "Add Discount",
+  "discount-form.add-conditions": "Add Conditions",
+  "discount-form.choose-a-condition-type": "Choose a condition type",
+  "discount-form.you-can-only-add-one-of-each-type-of-condition": "You can only add one of each type of condition",
+  "discount-form.you-cannot-add-any-more-conditions": "You cannot add any more conditions",
+  "discount-form.cancel": "Cancel",
+  "discount-form.save": "Save",
+  "add-condition-tables.cancel": "Cancel",
+  "add-condition-tables.save-and-add-more": "Save and add more",
+  "add-condition-tables.save-and-close": "Save and close",
+  "add-condition-tables.search-by-title": "Search by title...",
+  "add-condition-tables.search-groups": "Search groups...",
+  "add-condition-tables.search-products": "Search products...",
+  "add-condition-tables.search-by-tag": "Search by tag...",
+  "add-condition-tables.search-by-type": "Search by type...",
+  "details-condition-tables.search-by-title": "Search by title...",
+  "details-condition-tables.search-groups": "Search groups...",
+  "details-condition-tables.cancel": "Cancel",
+  "details-condition-tables.save-and-add-more": "Save and add more",
+  "details-condition-tables.save-and-close": "Save and close",
+  "details-condition-tables.search-products": "Search products...",
+  "details-condition-tables.search-by-tag": "Search by tag...",
+  "details-condition-tables.search-by-type": "Search by type...",
+  "edit-condition-tables.search-by-title": "Search by title...",
+  "edit-condition-tables.title": "Title",
+  "edit-condition-tables.search-groups": "Search groups...",
+  "edit-condition-tables.cancel": "Cancel",
+  "edit-condition-tables.delete-condition": "Delete condition",
+  "edit-condition-tables.save": "Save",
+  "edit-condition-tables.search-products": "Search products...",
+  "edit-condition-tables.search-by-tag": "Search by tag...",
+  "edit-condition-tables.search-by-type": "Search by type...",
+  "shared.title": "Title",
```

Comment:
I suggest using a consistent naming scheme for keys and not mix kebap-case with dot notation to increase interop with external parties which might choke on mixed naming schemes.

Given that JS and JSON use camelCase, camelCase seems advisable.

Reply:
But, i18next uses dot notation. Again, naming things is hard :D

Reply:
Camelcase is hard to read visually IMO 😞

Reply:
Yes, naming things is hard ^^ I just followed what i18next recommends 😄